Prox/Prox/Models/Place.swift

```diff
@@ -78,6 +79,7 @@ class Place: Hashable {
             self.wikipediaProvider = wikipediaProvider
             self.googleProvider = googleProvider
             self.customProvider = customProvider
+            self.isEvent = !(customProvider?.description ?? "").isEmpty
```

Comment:
nit: I'm not a fan of the readability of a bunch of syntax. I'd prefer if you used liuche's `String.isEmpty` method (the static version, not the instance one).